### PR TITLE
[Naitei][Task] Footer

### PR DESCRIPTION
**Pull request by TranMinhHuy2003**
## Related Tickets
[Code giao diện footer](https://edu-redmine.sun-asterisk.vn/issues/78466)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
footer_en
footer_vi
passrubocop_footer

### CODE REVIEW

**Comment by TranMinhHuy2003** [comment]
ready

File: app/assets/stylesheets/base.scss [hunk 1]
```diff
@@ -233,3 +233,122 @@ header {
...
   margin-right: 6px;
 }
+
+footer {
+  position: relative;
+}
+
+.footer__top {
+  padding: 40px 160px 14px 160px;
+  background-color: #1A1A1A;
```
Comment (SuKay712) [inline app/assets/stylesheets/base.scss]: quên sửa biến màu nè :v:

**Comment by TranMinhHuy2003** [comment]
fixed

File: app/views/layouts/_footer.html.erb [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        <li><%= t("footer.order_history") %></li>
+        <li><%= t("footer.cart") %></li>
+        <li><%= t("footer.wishlist") %></li>
+      </ul>
+    </div>
+
+    <div class="box">
+      <h2 class="box__header"><%= t("footer.help") %></h2>
+      <ul>
+        <li><%= t("footer.contact") %></li>
```
Comment (ghost) [inline app/views/layouts/_footer.html.erb]: bỏ bớt các () đi em

**Review by ghost — COMMENTED** [review]
cmt

**Comment by TranMinhHuy2003** [comment]
ready

**Review by ghost — COMMENTED** [review]
megred

**Pull request merged**